helix-view/src/editor.rs

```diff
@@ -66,7 +64,6 @@ impl Editor {
             theme_loader: themes,
             registers: Registers::default(),
             clipboard_provider: get_clipboard_provider(),
-            autoinfo: None,
```

Comment:
If you removed these how are you planning to show autoinfo?

Reply:
I moved it into `EditorView` and set it from there when handling key input.